content/docs/instrumenting/pushing.md

```diff
@@ -26,6 +26,56 @@ class.
 
  * For use from Ruby see the [Pushgateway documentation](https://github.com/prometheus/client_ruby#pushgateway).
 
+## Should I be using the Pushgateway?
```

Comment:
Could be, not sure. I guess I'm fine with either, but I guess it'd fit into the "Instrumentation" best practices doc (and linked from here)?